### PR TITLE
Additional documentation on credentialed CORS

### LINKED ISSUE #1865: Document Access-Control-Allow-Credentials in CORS docs

### PR DESCRIPTION
**Pull request by dvoytenko**
Closes #1865

### CODE REVIEW

File: spec/amp-cors-requests.md [hunk 1]
```diff
@@ -20,6 +20,21 @@ Many components and extensions in AMP take advantage of remote endpoints using C
...
+## CORS and Cookies
+
+Most of components that use CORS requests in AMP also either automatically set credentials mode or allow the
+author to optionally enable it. The credentials mode sends origin cookies in the CORS request and it will also
+allow the origin to set cookies in response (subject to 3rd-party cookie restrictions). The origin, however,
+must enable credentials mode on the server side using `Access-Control-Allow-Credentials: true` response header.
+
+### 3rd-Party Cookie Restrictions
+
+The normal browser 3rd-party cookie restrictions apply to the credetialed CORS requests. These restrictions depend
```
Comment (cramforce) [inline spec/amp-cors-requests.md]: nit: credentialed.

File: spec/amp-cors-requests.md [hunk 2]
```diff
@@ -20,6 +20,21 @@ Many components and extensions in AMP take advantage of remote endpoints using C
...
+on the browser and the platform, but most typically, the origin can only set cookies if the user has previously
```
Comment (cramforce) [inline spec/amp-cors-requests.md]: s/most typically/for some browsers

**Comment by cramforce** [comment]
LGTM with some comments

File: spec/amp-cors-requests.md (hunk 1 above)
Comment (dvoytenko) [inline spec/amp-cors-requests.md]: Done

File: spec/amp-cors-requests.md (hunk 2 above)
Comment (dvoytenko) [inline spec/amp-cors-requests.md]: Done

**Pull request merged**